src/js/state/Search/reducer.js

```diff
@@ -8,7 +8,8 @@ export const initSearchState: SearchState = {
   ],
   spanArgs: ["now - 5m", "now"],
   spanFocus: null,
-  space: "",
+  spaceID: "",
+  spaceName: "",
```

Comment:
I don't think we need both the id and the name here in the search reducer.

Reply:
Ah yeah, my first pass was to just include it everywhere but I suspect you're right that we don't need it in this. Will confirm and if we don't then will remove 👍

Reply:
Ah so we *do* need this. Or at least things are currently wired up to use that name for the tab title. which displays the space name and search terms

Reply:
I see, for their we can get the search.clusterId and the space.spaceId in the tab state to then look up spaces[clusterId][spaceId].spaceName in the root state.

If we keep that space name around, it will go out of date as soon as we allow for renaming the space. But if you want to revisit this when we add the space rename feature, I'm cool with that.